package.json

```diff
@@ -1,8 +1,50 @@
 {
-  "name": "icons",
+  "name": "@grafana/icons",
   "version": "1.0.0",
   "description": "Icons for Grafana",
   "repository": "https://github.com/grafana/icons.git",
   "author": "Grafana Labs",
-  "license": "Apache-2.0"
+  "license": "Apache-2.0",
+  "scripts": {
+    "icons:clean": "rm -rf ./src/icons",
+    "icons:create": "npx @svgr/cli ./svg",
```

Comment:
do you know if svgr sanitizes SVG for XSS at conversion? if not, can we make sure we do that with some static tool (so we avoid having to add some runtime dependency to the icons)